Lake/Util/Name.lean

```diff
@@ -16,87 +16,31 @@ export Lean (Name NameMap)
 
 namespace Name
 
-@[simp] theorem beq_rfl (n : Name) : (n == n) = true := by
-  simp only [BEq.beq]; induction n <;> simp [Name.beq, *]
-
-@[simp] theorem isPrefixOf_self {n : Name} : n.isPrefixOf n = true := by
-  cases n <;> simp [Name.isPrefixOf]
-
-@[simp] theorem isPrefixOf_append {n m : Name} : n.isPrefixOf (n ++ m) = true := by
-  simp only [HAppend.hAppend, Append.append]
-  induction m <;> simp [Name.isPrefixOf, Name.append, *]
```

Comment:
I think you may have accidently deleted these theorems.

Reply:
`beq_rfl` is redundant now.  The other ones are indeed an accident.